subgraphs/enzyme-core/mappings/v4/ExternalPositionManager.ts

```diff
@@ -2326,6 +2337,46 @@ export function handleCallOnExternalPositionExecutedForFund(event: CallOnExterna
   //   return;
   // }
 
+  if (type.label == 'ALICE') {
+    if (actionId == AliceActionId.PlaceOrder) {
+      let decoded = ethereum.decode('(tuple(uint16,bool,uint256,uint256)', event.params.actionArgs);
+
+      if (decoded == null) {
+        return;
+      }
+
+      let tuple = decoded.toTuple();
+
+      createAlicePositionChange(event.params.externalPosition, 'PlaceOrder', vault, event);
+    }
+
+    if (actionId == AliceActionId.RefundOrder) {
+      let decoded = ethereum.decode('tuple(uint256,uint16,bool,uint256,uint256,uint256)', event.params.actionArgs);
+
+      if (decoded == null) {
+        return;
+      }
+
+      let tuple = decoded.toTuple();
+
+      createAlicePositionChange(event.params.externalPosition, 'RefundOrder', vault, event);
+    }
+
+    if (actionId == AliceActionId.Sweep) {
+      let decoded = ethereum.decode('tuple(uint256[])', event.params.actionArgs);
```

Comment:
add tuplePrefixBytes otherwise it won't be decoded properly
```suggestion
      let decoded = ethereum.decode('tuple(uint256[])', tuplePrefixBytes(event.params.actionArgs));
```